lib/floe/workflow/intrinsic_function/transformer.rb

```diff
@@ -9,6 +9,11 @@ class Workflow
     class IntrinsicFunction
       class Transformer < Parslet::Transform
         OptionalArg = Struct.new(:type)
+        class Boolean
+          def self.to_s ; "Boolean" ; end
+          # NOTE: this is backwards
+          def self.kind_of?(value) ; value.kind_of?(TrueClass) || value.kind_of?(FalseClass) ; end
```

Comment:
I agree with Rubocop - not a fan of single-line methods, even if simple, so let's just blow this out.